pkg/tsdb/cloudwatch/time_series_query.go

```diff
@@ -121,6 +122,7 @@ func (e *cloudWatchExecutor) executeTimeSeriesQuery(ctx context.Context, logger
 			Error: fmt.Errorf("metric request error: %q", err),
 		}
 		resultChan <- &responseWrapper{
+			RefId:        getQueryRefIdFromErrorString(err.Error(), requestQueries),
```

Comment:
This is more of an improvement. We dont really need the refId in the error in order to display the error in the panel top left corner. However, we do need it to display it in the query editor inside the panel.